zap/src/test/java/org/zaproxy/zap/model/StandardParameterParserUnitTest.java

```diff
@@ -80,6 +82,53 @@ void defaultParser() {
         assertEquals(res2.get(3).getValue(), "g");
     }
 
+    @Test
+    void defaultTreePath() throws HttpMalformedHeaderException, URIException {
```

Comment:
Better use parameterized tests.

Reply:
Also be good to add one that has both query and form parameters.

Reply:
Added test cases using ParameterizedTest.